homeassistant/components/simplisafe/alarm_control_panel.py

```diff
@@ -94,6 +98,16 @@ def state(self):
         """Return the state of the entity."""
         return self._state
 
+    @property
+    def supported_features(self) -> int:
+        """Return the list of supported features."""
+        return SUPPORT_ALARM_ARM_HOME | SUPPORT_ALARM_ARM_AWAY
+
+    @property
+    def unique_id(self):
```

Comment:
This isn't necessary – it's already provided by the `SimpliSafeEntity` base class.